### PR TITLE
Add the new `Specify breaking` check to bors.toml

### PR DESCRIPTION
**Pull request by curquiza**
Should prevent this problem: https://github.com/meilisearch/milli/pull/489#issuecomment-1094988060

### CODE REVIEW

**Review by ManyTheFish — DISMISSED** [review]
bors merge

File: bors.toml [hunk 1]
```diff
@@ -4,5 +4,6 @@ status = [
     'Tests on windows-latest with stable',
     'Run Rustfmt',
 ]
+pr_status = ['Specify breaking']
```
Comment (curquiza) [inline bors.toml:7]:
```suggestion
pr_status = ['Enforce PR labels / Specify breaking']
```

**Comment by curquiza** [comment]
bors merge

**Comment by curquiza** [comment]
Ok definitely looks like it does not know the check, the naming is wrong

File: bors.toml [hunk 2]
```diff
@@ -4,5 +4,6 @@ status = [
     'Tests on windows-latest with stable',
     'Run Rustfmt',
 ]
+pr_status = ['Enforce PR labels / Specify breaking']
```
Comment (curquiza) [inline bors.toml:7]:
```suggestion
pr_status = ['Specify breaking']
```

**Review by MarinPostma — APPROVED** [review]

**Review by Kerollmops — APPROVED** [review]

**Comment by curquiza** [comment]
bors merge

**Comment by curquiza** [comment]
bors merge-

**Comment by curquiza** [comment]
bors merge

**Comment by curquiza** [comment]
bors merge

**Review by ManyTheFish — APPROVED** [review]

**Comment by curquiza** [comment]
bors merge

**Pull request merged**